meta-dts-distro/recipes-dts/dts/dasharo-deploy/dasharo-deploy

```diff
@@ -552,6 +552,15 @@ update() {
 
   handle_fw_switching $_can_switch_to_heads
 
+  if [ "$UPDATE_VERSION" == "1.1.1" ]; then
```

Comment:
@miczyg1 okay, now we check for the specific board [here](https://github.com/Dasharo/meta-dts/pull/108/commits/8607f9b0b2bfc1640d58c70b1e75883477e23f0f#diff-ac1fdd7af29e82bf8d7ca23462952d638564fce3189e36cbb6eebeb48f480d20R556-R559).

Reply:
I have said Z690, not Z790. You have added two extra boards. Z790 support 13th and 14th gen since the first release (i.e 0.9.0), Z790 does not have v1.1.1 yet.

Reply:
@miczyg1 sorry, fixed it [here](https://github.com/Dasharo/meta-dts/pull/108/commits/0e3b976f5e76a4616612b1a35e3edf39253f23bc)

Reply:
Possible board models are:
"PRO Z690-A WIFI DDR4(MS-7D25)"
"PRO Z690-A DDR4(MS-7D25)"
"PRO Z690-A WIFI (MS-7D25)"
"PRO Z690-A (MS-7D25)"

You haven't included DDR5 yet (the ones without DDR4 suffix).

Reply:
@miczyg1 added [here](https://github.com/Dasharo/meta-dts/pull/108/commits/b46bd633587240b388447b3fb1a5a348b0e66f02)